ws_objects/json.pbl.src/u_json.sru

```diff
@@ -75,32 +75,31 @@ public function string of_get_version ()
 public function u_json of_set_value (date ada_value) throws exception
 public function u_json of_set_value (datetime adt_value) throws exception
 public function u_json of_set_value (time at_value) throws exception
+public function u_json of_set_value (datetime adt_value, string as_format) throws exception
 end prototypes
 
 public subroutine of_load_file (string as_path) throws exception;/**
 description:	load a file and parse it
 parameters:		string as_path:	path where the json file will be found
 created:			2019-10-14
 author:			georg.brodbeck@informaticon.com
+2022-09-26:  Ramón San Félix Ramón add FileClose(li_file) rsrsystem.soft@gmail.com
 */
 
 string ls_json
 integer li_file
 
 li_file = fileopen(as_path, textmode!, read!, shared!)
 if li_file > 0 then
-	try
-		if filereadex(li_file, ls_json) < 0 then
-			throw of_get_exception('Error reading the file')
-		end if
-	finally
-		fileclose(li_file)
-	end try
+	if filereadex(li_file, ls_json) < 0 then
```

Comment:
Why did you remove try ... finally?
This would cause an open file handle in some cases.